src/main/java/org/terasology/inGameHelp/helpCategories/GeneralHelpCategory.java

```diff
@@ -59,11 +75,19 @@ public DocumentData getDocumentData() {
         return documentData;
     }
 
+    /**
+     * Currently does not do anything
```

Comment:
Stub `@Override` methods need to be NOT documented - that way they inherit the documentation of the interface that explains what the method _should_ do.